program/rust/src/tests/test_upd_price_v2.rs

```diff
@@ -43,13 +43,13 @@ fn test_upd_price_v2() -> Result<(), Box<dyn std::error::Error>> {
     let mut funding_setup = AccountSetup::new_funding();
     let funding_account = funding_setup.as_account_info();
 
```

Comment:
Note that this file is the same as `test_upd_price.rs` with some extra tests for pythnet, maybe worth merging them at some point and put a bunch of #[cfg(feature = "pythnet")] around